manifests/s/ServerData/Elevate/2.15.15/ServerData.Elevate.installer.yaml

```
@@ -0,0 +1,29 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ServerData.Elevate
+PackageVersion: 2.15.15
+InstallerLocale: en-US
+InstallerType: wix
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-02-12
+AppsAndFeaturesEntries:
+- DisplayVersion: 2.15.15.0
+  UpgradeCode: '{B7411227-AA11-4234-A03F-88494D4F714D}'

```

```suggestion
AppsAndFeaturesEntries:
- UpgradeCode: '{B7411227-AA11-4234-A03F-88494D4F714D}'
```
DisplayVersion is not necessary since all versions are treated as having infinitely trailing `.0`